src/replica/DirectorIndexJob.cc

```diff
@@ -388,18 +386,19 @@ void DirectorIndexJob::_loadDataIntoTable() {
         if (request == nullptr) break;
 
         // Load request's data into the destination table.
+        bool const localFile = true;
         try {
             string const query = g.loadDataInfile(
                     request->responseData().fileName, directorIndexTableName(database(), directorTable()),
                     controller()->serviceProvider()->config()->get<string>("worker", "ingest-charset-name"),
-                    localFile());
+                    localFile);
             h.conn->executeInOwnTransaction([&](auto conn) {
                 conn->execute(query);
                 // Loading operations based on this mechanism won't result in throwing exceptions in
                 // case of certain types of problems encountered during the loading, such as
                 // out-of-range data, duplicate keys, etc. These errors are reported as warnings
                 // which need to be retrieved using a special call to the database API.
-                if (localFile()) {
+                if (localFile) {
```

Comment:
Would it be better to remove the `if` statement?

Reply:
I too thought about removing the if. I will consider it.